### DISCUSSION TITLE
Upgrade bootstrap 5 based on AdminLTE v3.1

### DISCUSSION THREAD

**Discussion by CharlieYAN** [ColorlibHQ/AdminLTE#3814] [Q&A]
If I upgrade bootstrap from v.4 to v.5 on the AdminLTE v3.1, will it cause big problems on current components?
Can anyone evaluate the work? How many associated parts do I need to modify or possibly to be modified?

**Comment by REJack** [comment]
I would say yes, but I don't know if someone tested it.

**Reply by CharlieYAN** [reply]
I tried testing a little. It seems a lot variables and functions changed in Bootstrap 5. 
I feel it really not easy to make the upgrade building passed.  
Do you have good idea to make the upgrade steps clearer and eaiser?
